build.gradle.kts

```diff
@@ -20,7 +20,7 @@ plugins {
 }
 
 subprojects {
-    group = "io.github.lmos-ai.arc"
+    group = "ai.ancf.lmos.arc"
```

Comment:
Why not just ai.ancf.lmos?
And then have all artifacts under this group name?

Reply:
Sounds good to me! I've changed it.